plugins/org.eclipse.n4js/src/org/eclipse/n4js/scoping/ExportedElementsCollector.java

```diff
@@ -158,7 +159,8 @@ private void doCollectElements(AbstractNamespace namespace, CollectionInfo info)
 	private void doCollectElement(String exportedName, TExportableElement exportedElem, CollectionInfo info) {
 
 		boolean include = (info.includeHollows || !N4JSLanguageUtils.isHollowElement(exportedElem, variantHelper))
-				&& (info.includeVariables || !(exportedElem instanceof TVariable));
+				&& (info.includeVariables || !N4JSLanguageUtils.isValueOnlyElement(exportedElem, variantHelper));
```

Comment:
Would it make sense to now rename field `CollectionInfo#includeVariables` to `includeValues`? (also the corresponding parameter for method `#getExportedElements()`)